admin_delete.go

```diff
@@ -16,7 +16,11 @@ const (
 	pathUploadedRaws   = "/resources/raw/upload"
 )
 
-func (s *Service) bulkDeleteUploadedWithInvalidation(resourceType ResourceType, publicIDs []string) error {
+type HttpClient interface {
+	Do (req *http.Request) (*http.Response, error)
+}
+
+func (s *Service) bulkDeleteUploadedWithInvalidation(client HttpClient, resourceType ResourceType, publicIDs []string) error {
```

Comment:
I suggest putting the client on a service level and initializing it while servicing initialization.

Reply:
thanks! fixed